docs/dev/service_levels.md

```diff
@@ -30,7 +30,7 @@ SELECT * FROM  system.role_attributes WHERE role='r' and attribute_name='service
 ### Service Level Configuration Table
 
 ```
-    CREATE TABLE system_distributed.service_levels (
+    CREATE TABLE system.service_levels_v2 (
```

Comment:
Topology on raft is not mandatory yet, so I think it would be good to mention that this table is only used in that mode, and in the legacy topology mode we still use the old table.

I think that it is sufficient to only mention that the tables are different, and provide a link to the last section which goes into more detail about which tables were used in the legacy mode.